parameter/ParameterMgr.cpp

```diff
@@ -320,22 +320,10 @@ const CParameterMgr::SRemoteCommandParserItem CParameterMgr::gastRemoteCommandPa
 
 // Remote command parsers array Size
 CParameterMgr::CParameterMgr(const string& strConfigurationFilePath, log::ILogger& logger) :
-    _bTuningModeIsOn(false),
-    _bValueSpaceIsRaw(false),
-    _bOutputRawFormatIsHex(false),
-    _bAutoSyncOn(true),
     _pMainParameterBlackboard(new CParameterBlackboard),
```

Comment:
Do not move them, you are right. The new here needs to be warded. (BTW why is it not a member ?)